src/implementations/twig/components/hero-banner/README.md

```diff
@@ -0,0 +1,46 @@
+# ECL Hero Banner
+
+npm package: `@ecl/twig-component-hero-banner`
+
+```shell
+npm install --save @ecl/twig-component-hero-banner
+```
+
+### Parameters
+
+- **"type"** (string) (default: 'default') Type of banner (can be 'default','image','image-shade','primary')
```

Comment:
types seems to be now `white` `grey` `image` `image-gradient` `primary` `image-shade`, this was not kept up to date in ecl-twig..

Reply:
I'm not sure I understand your feedback here. I've taken the README file from the ecl-twig repo as well as its corresponding twig template file. If this line is somehow not matching your expectations, this probably means ecl-twig implementation should be updated in the other repo if we are to change something here.

Reply:
My expectations? These are the "types" that the component supports, not really a preference of mine. If you don't want to fix this here let's merge it like this, but it's wrong and it's hard to follow this kind of reasoning instead of simply doing the right thing.

Reply:
I like the the "doing the right thing", thank you for keeping this spirit.
I updated the files which I hope is done right.